lib/tasks/scripts/covid_medication_reminders.rb

```diff
@@ -1,10 +1,12 @@
 class CovidMedicationReminders
   def self.call(number_of_patients: 10_000)
-    if CountryConfig.current[:name] != "India" || !SimpleServer.env.production?
-      raise "Cannot send Covid medication reminders in this env"
+    raise "Experiments are not enabled in this env" unless Flipper.enabled?(:experiment)
```

Comment:
I think we should move this back to where it was (after the guard for country and environment). This will raise errors in Ethiopia, Bangladesh, and Sandbox, which seems like unnecessary noise.